frontend/src/assets/i18n/it.json

```
@@ -357,5 +357,755 @@
+                    "title": "Integrazioni Potenti",
+                    "description": "Connettiti con PMS, CRM e sistemi hospitality tramite API e webhook nativi."
+                }
+            }
+        },
+        "trusted": {
+            "eyebrow": "Scelto dai team di hospitality più moderni",
+            "title": "Dalle boutique hotel ai gruppi multi-struttura."
+        },
+        "testimonials": {
+            "eyebrow": "Cosa dicono i nostri clienti",
+            "title": "Affidato da leader hospitality worldwide.",
```

Biome (2.3.13)

[error] The key testimonials was already declared.

This is where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)